Ginger/GingerCoreCommon/UserCofig/UserProfile.cs

```diff
@@ -500,9 +500,12 @@ public string ProfileImage
             }
         }
 
+        private string mUserName;
+
         public string UserName
         {
-            get { return Environment.UserName; }
+            get { return string.IsNullOrEmpty(mUserName) ? Environment.UserName : mUserName; }
```

Comment:
why are we changing username in userprofile.cs. its being used in sourcecontrol?
i believe its used to display username in top right corner and userprofilepage